fork-clone-workflow.md

```diff
@@ -33,9 +35,61 @@ own version of the https://github.com/mindfulness-at-the-computer/git-info reosi
-for the project. 
-### Work in progress (taking a snapshot of work on 2/27/18)
+for the project. You will [clone](git-glossary.md#clone) your fork on to your development machine. 
+Then you will configure your clone so that it knows how to find the main (organizational) repository
+from which the fork was created.
+
+* Change directory to the location on your development machine where you want to store the clone. You can clone
+to any location, but it is a good idea to keep your clones in a fixed location. In this example `joeuser` keeps
+cloned repositories in the `/opt/repos` directory.
+```bash
+cd /opt/repos
+```
+* Clone your fork using the `git clone` command. You can find the URL of your fork on the main [gihub](git-glossary.md#github)
+page of your fork. Click on the `Clone or download` button and a popup will show you the URL to use if you want to
+`Clone with SSH`. The SSH option is the easiest option to use but requires that you have confiured configured your
+[github](git-glossary.md#github) account with your public SSH key. As an alternative, you may select `Use HTTPS`. 
+The HTTPS option uses a different URL and will require you to supply your [github](git-glossary.md#github)
+account password each time you access your fork. The SSH option does not require you to supply your [gihub](git-glossary.md#github)
+when accessing your fork.
+```bash
+git clone git@github.com:joeuser/git-info.git
+```
+* After executing the `git clone` command successfully, there will be a directory containing a clone of your fork.
+```bash
+$ ls -d git-info
+git-info
+```
+* Change directory to the newly create reporitory directory (`git-info` in this example) to configure the local clone so that it
+knows where to find the remote repository from which your fork was created. Your git clone maintains a set of
+[references](git_glossary.md#references) to remote repositories. 
+git automatically creates the `origin` reference when the clone is created. The `git remote -v` command lists the known
+[references](git_glossary.md#references) and the values associated with each reference.
+```bash
+$ cd git-info
+$ git remote -v
+origin  git@github.com:joeuser/git-info.git (fetch)
+origin  git@github.com:joeuser/git-info.git (push)
+```
+* By convention we use a reference named `upstream` to point to the remote repository from which your fork was 
+created. We need to define the `upstream` remote reference by supplying the correct URL. You can find the 
+URL of the repository from which you forked by clicking the link next to `forked from` label just below the name of your
+fork on your fork's main [github](git-glossary.md#github) page. In this example, the fork was cloned from the 
+`mindfulness-at-the-computer` main repository. Below is the git commnand used to define the `upstream` reference.
+```bash
+$ git remote add upstream git@github.com:mindfulness-at-the-computer/git-info.git
```

Comment:
This is what i use myself: `git remote add upstream https://github.com/mindfulness-at-the-computer/mindfulness-at-the-computer.git` - Maybe this is something that is different depending on if we are using ssh or git? (I've never used ssh for git repos myself so i'm just guessing)

Reply:
Yes, the URL format is different for SSH versus HTTPS. If we change to documenting HTTPS as the default, as mentioned above, I will change this example.

Reply:
My 2c... As I said on Gitter, I use SSH for repos I can (and intend to) push to. Otherwise I use https. There is no issue with mixing and matching https and ssh remotes.